webpack.config.js

```diff
@@ -171,7 +171,8 @@ module.exports = {
   optimization: {
     splitChunks: {
       chunks: 'async'
-    }
+    },
+    minimize: true
```

Comment:
I think minimization is not necessary in `development` mode and `production` already has it by default. `optimization` is a rather annoying option since there is not an effective way to do it